docs/source/model_design/vivarium_model_components/fertility/index.rst

```diff
@@ -0,0 +1,149 @@
+.. _fertility_live_births:
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  ---------------------
+
+  Section Level 2 (#.#)
+  +++++++++++++++++++++
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+==============================================
+Fertility Component that uses Live Births data
+==============================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Overview
+-------------
+
+This document is intended to represent the fertility model for a simulation that adds new simulants based on live births data,
+rather than by identifying which simulants give birth.
+
+GBD Modeling Strategy
+----------------------
+
+Pregnancy and births are not explicit outcomes in the GBD study. However, there are location- and year-specfic GBD covariates related to fertility and births which we can use to calculate the number
+of new simulants to add per timestep.
+
+.. list-table:: Covariates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Covariate name
+    - Covariate ID
+    - Note
+  * - Live births by sex
+    - 1106
+    - 
+
+
+Vivarium Modeling Strategy
+----------------------------
+
+In a simulation where we do not model births by identifying which simulants give birth, we must instead use live births data
+to determine how many simulants to add in each time step.
+We will scale down the live births of the total GBD population by the same "model scale"  we use in our simulation.
+
+While we always scale down total GBD population live births when **adding** simulants,
+the number of simulants we **initialize** may be less than the scaled down total GBD population due to 
+restrictions on variables such as age.
+For example, for a child nutrition simulation, we may only simulate children under 5.
+
+Considering both model scale and population restrictions, we can define four different populations 
+which will be useful in this discussion:
+
+  * Restricted GBD (full-scale)  
+  * Unrestricted/total GBD (full-scale) 
+  * Restricted "simulated" (model-scale)
+  * Unrestricted/total "model-implied" (model-scale)
+
+Note that we differentiate model-scale populations as either "simulated" or "model-implied" 
+depending on whether they are restricted or not. 
+"Simulated" reflects that the restiction defines the population that we create simulants for.
```

Comment:
I'm concerned that this term doesn't adequately capture that we could e.g. initialize simulants under 5 but then continue tracking them into adulthood. Clearly people over 5 in that case would be "simulated," just not _initialized_.

Though, one way to look at it then is that our population restriction is actually "cohorts born after 2018" or similar. We simulate those cohorts, but not older cohorts, so "simulated" makes sense. But I think this framing might require some notes elsewhere in the page.